godot_project/autoloads/data/periodic_table_data.csv

```diff
@@ -5,23 +5,23 @@ Number,Period,Group,Symbol,Name,Mass,Radius,van der Waals Radii,O1 Covalent Radi
 4,2,2,Be,Beryllium,9.012,105,153,102,90,85,2,1,1551,3750,1.85,#d6fb00,#bebebe,#d6fb00,#000000,0
 5,2,13,B,Boron,10.81,85,192,85,78,73,3,2,2349,4200,2.46,#ffcecd,#bebebe,#ffcecd,#000000,0
 6,2,14,C,Carbon,12.011,70,170,75,67,60,4,2,3820,5100,3.51,#190039,#7b268e,#692079,#ffffff,0
-7,2,15,N,Nitrogen,14007,65,155,71,60,54,5,2,63.1,77.4,1.17,#0000bf,#bebebe,#0000bf,#ffffff,1
+7,2,15,N,Nitrogen,14.007,65,155,71,60,54,5,2,63.1,77.4,1.17,#0000bf,#bebebe,#0000bf,#ffffff,1
 8,2,16,O,Oxygen,15.999,60,152,63,57,53,6,3,54.4,90.2,1.33,#870604,#b10b07,#b10b07,#ffffff,3
 9,2,17,F,Fluorine,18.998,50,147,64,59,53,7,1,53.5,85,1.58,#e1f262,#bebebe,#e1f262,#000000,0
 10,2,18,Ne,Neon,20.18,38,154,67,96,,8,3,24.6,27.1,0.84,#c6ecf8,#bebebe,#c6ecf8,#000000,0
 11,3,1,Na,Sodium,22.99,180,227,155,160,,1,1,371,1156,0.97,#c48af6,#bebebe,#c48af6,#000000,0
-12,3,2,Mg,Magnesium,24305,150,173,139,132,127,2,3,923,1380,1.74,#a6fa00,#bebebe,#a6fa00,#000000,0
+12,3,2,Mg,Magnesium,24.305,150,173,139,132,127,2,3,923,1380,1.74,#a6fa00,#bebebe,#a6fa00,#000000,0
 13,3,13,Al,Aluminium,26.92,125,184,126,113,111,3,1,933,2740,2.7,#e1c0bf,#bebebe,#e1c0bf,#000000,0
 14,3,14,Si,Silicon,28.085,110,210,116,107,102,4,3,1683,2628,2.33,#a2b6b7,#3d6778,#3d6778,#000000,0
-15,3,15,P,Phosphorus,30974,100,180,111,102,94,5,1,317,550,1.82,#f89c00,#b57c03,#b57c03,#000000,0
-16,3,16,S,Sulfur,62.065,100,180,103,94,95,6,4,338,718,2.06,#faf700,#966002,#966002,#000000,2
+15,3,15,P,Phosphorus,30.974,100,180,111,102,94,5,1,317,550,1.82,#f89c00,#b57c03,#b57c03,#000000,0
+16,3,16,S,Sulfur,32.065,100,180,103,94,95,6,4,338,718,2.06,#faf700,#966002,#966002,#000000,2
```

Comment:
leaving a note for you just to ensure this one is correct, since it stands out, all other ones had period at wrong place, this one after a change is using totally different digits

Reply:
yeah, 32.065 is correct